rootfs/manager/Makefile

```diff
@@ -11,30 +11,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-DOCKER_REGISTRY := gcr.io
-PREFIX := $(DOCKER_REGISTRY)/$(PROJECT)
-IMAGE ?= manager
-TAG ?= git-$(shell git rev-parse --short HEAD)
-FULL_IMAGE := $(PREFIX)/$(IMAGE)
 
+IMAGE ?= manager
 KUBE_VERSION ?= v1.1.7
 
-.PHONY: container
-container: binary kubectl
-	docker build -t $(FULL_IMAGE):latest -f Dockerfile .
-	docker tag $(FULL_IMAGE):latest $(FULL_IMAGE):$(TAG)
-
-.PHONY: push
-push: container
-ifeq ($(DOCKER_REGISTRY),gcr.io)
-	gcloud docker push $(PREFIX)/$(IMAGE):$(TAG)
-else
-	docker push $(PREFIX)/$(IMAGE):$(TAG)
-endif
+include ../include.mk
 
-binary:
-	cp ../../bin/linux-amd64/manager ./bin
+.PHONY: extras
+extras: kubectl
```

Comment:
I don't follow you there. This is a make target that just goes to another make target, and it's in a top-level file, not a shared one.